src/components/anonymous/Login.js

```diff
@@ -118,7 +118,8 @@ function Login() {
             <h2>Connexion</h2>
             <div>
               <div>
-                {(error && !error?.resetPasswordCnil) && <span style={{ color: 'red' }}>{error.error ? error.error : 'Une erreur s\'est produite'}</span>}
+                {(error && !error?.resetPasswordCnil && !error?.openPopinVerifyCode) &&
+                <span style={{ color: 'red' }}>{error.error ? error.error : 'Une erreur s\'est produite'}</span>}
```

Comment:
```suggestion
                <span style={{ color: 'red' }}>{error.error ?? 'Une erreur s\'est produite'}</span>}
```
Mais d'ailleurs je ne suis même pas certain car on avait pas des cas error.error.message.toString() à d'autres endroits ? je crois que error.error est un object par moment

Reply:
A dire vrai je ne sais pas c'est de l'existant et ça à l'air d'afficher "Une erreur s\'est produite" pour toutes les erreurs non ?